### PR TITLE
New class

### PR DESCRIPTION
**Pull request by roselineshapi**
The class will be used to interact with the Datastore
Timestamp is the time when an entity is made.

### CODE REVIEW

**Comment by roselineshapi** [comment]
added a new class

File: sps-team-38/src/main/webapp/script.js [hunk 1]
```diff
@@ -18,4 +18,24 @@ function onSignIn(googleUser){
...
+}
+
+
+function createListElement(text){
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
+
+
```
Comment (alexschles) [inline sps-team-38/src/main/webapp/script.js]: This is going to cause a merge conflict with my pull request from last night b/c I edited the same file

Comment (roselineshapi) [reply sps-team-38/src/main/webapp/script.js]: So we have to merge yours first? Or

Comment (eknuviad) [reply sps-team-38/src/main/webapp/script.js]: I think so.

Comment (eknuviad) [reply sps-team-38/src/main/webapp/script.js]: I'll merge it to see.

Comment (eknuviad) [reply sps-team-38/src/main/webapp/script.js]: Resolved the merge conflict

**Comment by alexschles** [comment]
LGTM

**Review by eknuviad — APPROVED** [review]
LGTM

**Pull request merged**